src/lib/core/backtrace.cc

```diff
@@ -246,11 +249,11 @@ __asm__ volatile(
 	"\tmovq 40(%%rsp), %%rbp\n"
 	/* Set first arg and call */
 	"\tmovq %0, %%rdi\n"
-#ifdef TARGET_OS_DARWIN
-	"\tandq $0xfffffffffffffff0, %%rsp\n"
-#endif
+	".cfi_remember_state\n"
+	".cfi_def_cfa %%rsp, 8 * 7\n"
 	"\tleaq %P2(%%rip), %%rax\n"
 	"\tcall *%%rax\n"
+	".cfi_restore_state\n"
 	/* Restore old sp and context */
 	"\tmov %%rax, %%rsp\n"
 	"\tpopq %%r15\n"
```

Comment:
Side note: Why there is no same change for __i386? Do we declare that x86 (not x64) is unsupported?

Reply:
Yes, according to our README.